web/auth/passphrase.go

```diff
@@ -133,11 +134,12 @@ func passphraseRenew(c echo.Context) error {
 		return c.Redirect(http.StatusSeeOther, redirect)
 	}
 	pass := []byte(c.FormValue("passphrase"))
+	iterations, _ := strconv.Atoi(c.FormValue("iterations"))
```

Comment:
Est-ce que ça ne pose pas problème d'utiliser une autre fonction de conversion que `strconv.ParseInt(<str>, 10, 64)` utilisée dans le reste du code pour les itérations ?

Reply:
En fait, on utilise `Atoi` quand on veut un `int` et `ParseInt` quand on veut un `int64`. Et je devrais aussi utiliser `Atoi` dans l'autre cas.